pyproject.toml

```diff
@@ -63,6 +64,7 @@ pytest-memray = {version = "^1.0.0", platform = "linux"}
 python-docx = "^0.8.11"
 boto3 = "^1.23.10"
 boto3-stubs = {extras = ["essential"], version = "^1.24.17"}
+botocore = "^1.34.141"
```

Comment:
was this added by poetry? why do we need it?

Reply:
I added it to get botocore to a version that doesn't clash with advocate. Both have a urllib3 dependency; without specifying a newer botocore version, advocate wouldn't get installed.